subsys/net/lib/shell/CMakeLists.txt

```diff
@@ -37,3 +37,4 @@ zephyr_library_sources_ifdef(CONFIG_NET_SHELL_UDP_SUPPORTED udp.c)
 zephyr_library_sources_ifdef(CONFIG_NET_SHELL_VIRTUAL_SUPPORTED virtual.c)
```

Comment:
Please change the commit subject of `drivers: net: add qbv shell commands` to
`net: shell: Add Qbv shell`